superset/assets/src/SqlLab/actions/sqlLab.js

```diff
@@ -785,8 +786,17 @@ export function queryEditorSetSchemaOptions(queryEditor, options) {
   return { type: QUERY_EDITOR_SET_SCHEMA_OPTIONS, queryEditor, options };
 }
 
-export function queryEditorSetTableOptions(queryEditor, options) {
-  return { type: QUERY_EDITOR_SET_TABLE_OPTIONS, queryEditor, options };
+export function queryEditorSetTableOptions(
```

Comment:
in general, i think it's best practice to make the function name match the action type. let's correct that?

Reply:
True, will adjust accordingly.